src/config/config.ts

```diff
@@ -1,3 +1,5 @@
+// TODO we need to build this into CI so that we don't have to remember to do this....
```

Comment:
this seems weird... did i do this? 🤔

Reply:
haha... yea... Is there something supposed to be done for this too?

Reply:
there's no doc store in this repo soo....

Reply:
I have removed this file in https://github.com/Open-Attestation/token-registry/pull/32/commits/34846d4344e83c572ff1106228fc71bb139582e4 because even the `TitleEscrowCreator` doesn't seem relevant now that we use the cloner.